e2e/src/orbs-client.ts

```diff
@@ -57,25 +55,29 @@ export class OrbsClientSession {
   }
 
   async call(contractAddress: string, payload: string) {
-    const { resultJson } = await this.orbsClient.call({
+    const { resultJson } = await this.orbsClient.callContract({
       sender: this.getAddress(),
-      contractAddress: contractAddress,
+      contractAddress: {address: contractAddress},
```

Comment:
Why do we need another level of indirection here? I.e., why not `contractAddress,`?

Reply:
It's not. It was just an abstraction layer I added as an interim step towards a new structure, while still keeping the existing addressing scheme